File: backend/src/main/java/com/github/fjbaldon/attendex/backend/dto/EventAttendeeSyncResponse.java
package com.github.fjbaldon.attendex.backend.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class EventAttendeeSyncResponse {
    private Long attendeeId;
    private String uniqueIdentifier;
    private String qrCodeHash;
    private String firstName;
    private String lastName;
}

File: capture/app/src/main/java/com/github/fjbaldon/attendex/capture/core/data/remote/EventAttendeeSyncResponse.kt
package com.github.fjbaldon.attendex.capture.core.data.remote

import kotlinx.serialization.Serializable

@Serializable
data class EventAttendeeSyncResponse(
    val attendeeId: Long,
    val identity: String,
    val qrCodeHash: String,
    val firstName: String,
    val lastName: String,
)
